### PR TITLE
Minor updates

### PR DESCRIPTION
**Pull request by joinimran**
As requested by user, I have updated the log defining path and logs location.

Problem: https://github.com/MicrosoftDocs/windows-itpro-docs/issues/5023

### CODE REVIEW

File: devices/surface/surface-dock-firmware-update.md [hunk 1]
```diff
@@ -73,11 +74,12 @@ Successful completion of Surface Dock Firmware Update results in new registry ke
 
 ## Event logging
 
+
```
Comment (illfated) [inline devices/surface/surface-dock-firmware-update.md]: Possibly redundant blank line?
```suggestion
```
(or is it needed for some unexpected reason?)

File: devices/surface/surface-dock-firmware-update.md [hunk 2]
```diff
@@ -73,11 +74,12 @@ Successful completion of Surface Dock Firmware Update results in new registry ke
...
 
 | Log                              | Location                               | Notes                                                                                                                                                                                                         |
 | -------------------------------- | -------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| Surface Dock Firmware Update log | /l*v %windir%\logs\ SurfaceDockFWI.log | Earlier versions of this tool wrote events to Applications and Services Logs\Microsoft Surface Dock Updater.                                                                                                  |
+| Surface Dock Firmware Update log | /l*v %windir%\logs\Applications\ SurfaceDockFWI.log | Earlier versions of this tool wrote events to Applications and Services Logs\Microsoft Surface Dock Updater.                                                                                                  |
```
Comment (illfated) [inline devices/surface/surface-dock-firmware-update.md]: I think we should avoid spaces in a path that may be copy-pasted by users.
```suggestion
| Surface Dock Firmware Update log | /l*v %windir%\logs\Applications\SurfaceDockFWI.log | Earlier versions of this tool wrote events to Applications and Services Logs\Microsoft Surface Dock Updater.                                                                                                  |
```
(yes, I know it was there already before this PR, but we do not need to keep a bug when we have found it) 😉

**Comment by joinimran** [comment]
Good to go.

**Comment by illfated** [comment]
Rich Text Diff view, for easier overview of the changes:
https://github.com/MicrosoftDocs/windows-itpro-docs/pull/5093/files?short_path=46e8921#diff-46e8921ff4540914a5b23145a04ec20b

**Comment by joinimran** [comment]
Any input is required from my side?

**Comment by illfated** [comment]
Not that I know of. We just need follow-up from the Docs team.

@mypil : Will you get in touch with the doc author & doc manager?

**Comment by mypil** [comment]
@Dansimp - PR has been copy edited and is ready for final review. This is based on issue #5023

Thank you.

cc: @kenwith

**Pull request merged**